views/relations.py

```diff
@@ -0,0 +1,52 @@
+from flask import request
+from flask.views import MethodView
+from controllers.relations import RelationsController
+
+import json
+from datetime import datetime
+
+class RelationsView(MethodView):
+
+    @classmethod
+    def make_relation(cls):
+        data = json.loads(request.data.decode('utf-8'))
+        req_params = ['user_id', 'first', 'last', 'birth_year', 'death_year', 'is_deceased', 'gender', 'relation', 'notes', 'is_step', 'is_adopted', 'birth_date', 'lives_in', 'nickname']
+        for param in req_params:
+            if param not in data:
+                return json.dumps({"response": "ill-formed request"}), 400
+
+        if not (isinstance(data['user_id'], int) and isinstance(data['first'], str) and isinstance(data['last'], str) and isinstance(data['is_deceased'], bool) and isinstance(data['gender'], str) and isinstance(data['relation'][0], str) and isinstance(data['relation'][1], int) and isinstance(data['notes'], str) and isinstance(data['is_step'], bool) and isinstance(data['is_adopted'], bool) and isinstance(data['lives_in'], str) and isinstance(data['nickname'], str)):
```

Comment:
view should be marshaling the input. Checking the type leaks abstraction of the model. I remember telling y'all to do this, but I figured that I shouldn't. Just move this to model.